templates/layout.html.twig

```diff
@@ -82,20 +82,22 @@ Enjoy your theme!
             <div class="page-header d-print-none">
                 <div class="row align-items-center">
                     <div class="col">
+                        {% block breadcrumb %}
                         <div class="page-pretitle">
-                            {% block page_subtitle %}Overview{% endblock %}
+                            {% block page_pretitle %}Overview{% endblock %}
                         </div>
+                        {% endblock %}
                         <h2 class="page-title">
                             {% block page_title %}Dashboard{% endblock %}
                         </h2>
-                        {% block page_stats %}
+                        {% block page_stats %}{% block page_subtitle %}
                         <div class="text-muted mt-1">
                             1-10 of 100
                         </div>
-                        {% endblock %}
+                        {% endblock %}{% endblock %}
                     </div>
                     <div class="col-auto ms-auto d-print-none">
-                        {% block breadcrumb %}
+                        {% block page_header_actions %}
```

Comment:
The other two are called `header_X`, then lets call this one `header_actions` as well, to be consistent

Reply:
IDK if it's a good idea, It's inside of `{% block page_header %}`.

With PR https://github.com/kevinpapst/TablerBundle/pull/3:
- `<header>` has block `header_id`.
- `.page-header` has block `page_header_id`.

At the end, `header_actions` sounds like action in the menu.
`page_header_X` relate to the header of the current page.

Reply:
Then rename the other ones?

Reply:
That is the question 😉 
> The other two are called header_X

Who are the two ? In `main` branch or PR ?

Reply:
From what I can see the block names are: `page_pretitle`, `page_stats`, `page_title`, `page_subtitle`, `page_header_actions` and `breadcrumb`. 
<img width="357" alt="Bildschirmfoto 2022-01-04 um 14 10 43" src="https://user-images.githubusercontent.com/533162/148064055-af93b9e1-3fe1-4d4e-ba3f-6436472bb733.png">

We can most certainly find a consistent naming, can't we? Now that we change it once and for all.

Reply:
I see now,
We can call them `page_X` when it depend on a specific page. 👍 

At the end we can:
- Leave like that: `page_pretitle`, `page_stats`, `page_title`, `page_subtitle`.
- Remove `page_stats`.
- And change `page_header_actions` to `page_actions`.

But now you make me question about `breadcrumb` 😄
Should we call him `page_breadcrumb` ? It will be a BC Break for you..
`breadcrumb` simple is fine to me.

Reply:
I don't care about the BC breaks on my end. It is better to make it correct ones and then never have to touch it again. 

Hm, `breadcrumb` sounds correct to me as well, but with the same argument we could call it `actions` as well.
The other blocks are called `page_content_before `, `page_content` ... so we should either remove `page_` from all of them (brutal BC break, please not) or prefix the breadcrumb as well to `page_breadcrumb`. 

Honestly: I haven't thought about it before, but now that we discuss I'd like to have it consistent.

Reply:
`page_content_before` , `page_content`, ... is not a problem to me.
Once again, there are targeting `page` configuration.
Like `page_wrapper_id` and `.page-body`. 
It's fine to me.

To conclude, I would leave `breadcrumb` as it, and let `page_actions` too.
But if you want to rename `page_actions` into `actions`, it is all yours.

Reply:
```suggestion
                        {% block page_actions %}
```